adafruit_lps2x.py

```diff
@@ -139,31 +148,36 @@ class LPS2X:  # pylint: disable=too-many-instance-attributes
     _raw_temperature = ROUnaryStruct(_LPS2X_TEMP_OUT_L, "<h")
     _raw_pressure = ROBits(24, _LPS2X_PRESS_OUT_XL, 0, 3)
 
-    def __init__(self, i2c_bus, address=_LPS2X_DEFAULT_ADDRESS, chip_id=None):
+    def __init__(
+        self,
+        i2c_bus: I2C,
+        address: int = _LPS2X_DEFAULT_ADDRESS,
+        chip_id: Optional[int] = None,
```

Comment:
See the previous review regarding changes to fix handling `chip_id` usage.

Reply:
yep missed it totally, fixed now....

Reply:
I don't think this change has been pushed yet, it's not currently in the pull request changes

Reply:
you are correct ... pushing the commit now